### ISSUE TITLE
Multilayer Perceptron class with full interface design.

### ISSUE THREAD

**Issue by Siemashko** [Siemashko/DL-Project-1#1]
Must include all required methods - (__init__, fit, predict) with specified data types for parameters and results.

required MLP parameters:
* number of hidden layers and number of neurons in hidden layers 
* random seed
* activation function
* bias presence
* batch size
* number of iterations
* learning rate
* momentum
* problem type:  classification or regression
* loss function
